### PR TITLE
updated UI of footer

### PR DESCRIPTION
**Pull request by whiteHatpro**
fixed issue #16

### CODE REVIEW

**Comment by whiteHatpro** [comment]
@anushbhatia please review the pr

**Review by anushbhatia — CHANGES_REQUESTED** [review]
don't add package-lock.json
just check the .gitignore if that file is ignored

**Review by anushbhatia — COMMENTED** [review]
also remove unused background images that are added.
A suggestion food related image would add a better look to the current website

**Comment by whiteHatpro** [comment]
@anushbhatia ok will do it asap

**Comment by whiteHatpro** [comment]
@anushbhatia i have done the changes as requested by you

File: package-lock.json [hunk 1]
```diff
@@ -1305,6 +1305,12 @@
         }
       }
     },
+    "@gar/promisify": {
+      "version": "1.1.2",
+      "resolved": "https://registry.npmjs.org/@gar/promisify/-/promisify-1.1.2.tgz",
```
Comment (smaranjitghose) [inline package-lock.json]: Remove this file from your commit please

File: src/App.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
-import "./App.css";
-import Home from "./components/Home";
-import GlobalStyle from "./globalStyles";
-import Footer from "./components/Footer";
+
```
Comment (smaranjitghose) [inline src/App.js]: Avoid unnecessary formatting  changes like black spaces as much as possible

File: src/scss/_colors.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
+// color variables used in the app.
```
Comment (smaranjitghose) [inline src/scss/_colors.scss:1]: Nice work segregating this!

File: src/githublogo/github.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" class="feather feather-github" viewBox="0 0 24 24"><path d="M9 19c-5 1.5-5-2.5-7-3m14 6v-3.87a3.37 3.37 0 0 0-.94-2.61c3.14-.35 6.44-1.54 6.44-7A5.44 5.44 0 0 0 20 4.77 5.07 5.07 0 0 0 19.91 1S18.73.65 16 2.48a13.38 13.38 0 0 0-7 0C6.27.65 5.09 1 5.09 1A5.07 5.07 0 0 0 5 4.77a5.44 5.44 0 0 0-1.5 3.78c0 5.42 3.3 6.61 6.44 7A3.37 3.37 0 0 0 9 18.13V22"/></svg>
```
Comment (smaranjitghose) [inline src/githublogo/github.svg]: Should be inside assets

File: src/components/Footer/footer.module.scss [hunk 5]
```diff
@@ -0,0 +1,68 @@
+@use '../../scss/colors';
+
+.footerDiv {
+	position: relative;
+	margin-top: 2rem;
+	background-image: url("./footer/brooke.jpg");
```
Comment (smaranjitghose) [inline src/components/Footer/footer.module.scss]: should be present inside assets

File: src/components/Footer/Footer.jsx [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+						loop={true}
+						strokeColor="#ffffff"
+						speed={1}
+					/>
+				</a>
+			</div>
+			<div>
+				<Fade>
+					<p className={styles.footerText}>
+						{emoji("Made with ❤️ in India for the students of the world.")}
```
Comment (smaranjitghose) [inline src/components/Footer/Footer.jsx]: Made with ❤️ in India for the foodies around the globe

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
Change the text color to make it distinct from the background image please

**Comment by whiteHatpro** [comment]
@smaranjitghose done the changes

**Comment by whiteHatpro** [comment]
@smaranjitghose @anushbhatia its working fine in my local machine
i don't know why there is an deployment issue.

**Comment by anushbhatia** [comment]
> @smaranjitghose @anushbhatia its working fine in my local machine i don't know why there is an deployment issue.

is there any warnings when you run the project do eliminate them if there are any?

**Comment by whiteHatpro** [comment]
> > @smaranjitghose @anushbhatia its working fine in my local machine i don't know why there is an deployment issue.
> 
> is there any warnings when you run the project do eliminate them if there are any?

@anushbhatia there is no warnings and it is compiled successfully

**Comment by whiteHatpro** [comment]
@anushbhatia please look at the terminal

Screenshot (113)

Screenshot (114)

**Comment by anushbhatia** [comment]
> @anushbhatia please look at the terminal
> 
> Screenshot (113)
> 
> Screenshot (114)

recreate this pr with latest code

**Pull request closed without merging**